pulp_rpm/app/depsolving.py

```diff
@@ -899,9 +899,9 @@ def run_solver_jobs(jobs):
                 unit_install_job = self._pool.Job(flags, solvable.id)
                 install_jobs.append(unit_install_job)
 
-        # Depsolv using the list of unit install jobs, add them to the results
-        solvables_copied = run_solver_jobs(install_jobs)
-        result_solvables.update(solvables_copied)
+            # Depsolve using the list of unit install jobs, add them to the results
+            solvables_copied = run_solver_jobs(install_jobs)
```

Comment:
Hmm, no, I don't see why that would be the case?  On either branch, the install job for the solvable that was popped is added to the list (lines 891 and 900), and on the module branch it includes all the artifacts as well.

Reply:
Ah I see, I wasn't reading the flow closely enough, thanks

Reply:
It is true that it will run ```run_solver_jobs``` for each individual package being copied now, whereas before it was running it for all of them at once.   That's what it did in Pulp 2 as well and what I was told by Igor Gnatenko it probably should do.  I think by "installing" both of them at once it picks 5.21 which has no dependencies and neglects 0.71 entirely.  There might be some hacks to get around that but I'd rather stick to what we know works until we have enough tests to start playing around with different ways to accomplish this with less overhead.

Reply:
Concur, I recall that discussion in the Pulp2 context